wallet/wallet.go

```diff
@@ -653,6 +664,9 @@ func (w *Wallet) recovery(chainClient chain.Interface,
 	if err != nil {
 		return err
 	}
+	log.Debugf("Chain backend has best block %d which is current: %v",
+		bestHeight, chainClient.IsCurrent())
+	chainClient.NotifyBlocks()
```

Comment:
why is this change needed?

Reply:
Oops, that was some debugging/trial-and-error code I committed by accident. Removed.